.github/actions/build-dotnet/action.yml

```diff
@@ -46,18 +46,26 @@ inputs:
     required: false
     default: "*.sln"
     description: >
-      The path of the solution file to be built. If you have exactly one .sln file in the current directory then this 
+      The path of the solution file to be built. If you have exactly one .sln file in the current directory then this
       can be omitted. Otherwise make sure to specify it to avoid an "MSB1008: Only one project can be specified." error
-      that is thrown when the `*.sln` wildcard results in multiple files. The value is given to PowerShell's 
+      that is thrown when the `*.sln` wildcard results in multiple files. The value is given to PowerShell's
       `Get-ChildItem` cmdlet as-is, so grepping is still possible but a name with spaces must be escaped separately.
+  powershell-analyzer-path:
+    required: false
+    default: "./tools/Lombiq.Analyzers.PowerShell/Invoke-Analyzer.ps1"
+    description: >
+      The path of the "Invoke-Analyzer.ps1" file to be used for linting PowerShell script files. If not found, nothing
+      happens. So if you have no use for it, feel free to ignore.
```

Comment:
This is not related to .NET build. It can be part of the OC or .NET build workflow because we commonly use PS scripts in those projects too, but not in the .NET build action.

Instead, we can have a PS analyzer action and use that from the workflows.

Note the necessity to also update the PowerShell-Analyzers Readme related to this.

Reply:
My thought process was that we do static code analysis before build.

Reply:
We can do that, but PS analysis is unrelated to the .NET build, thus it shouldn't be in this action.

Reply:
It's removed now due to using MSBuild only.